python/GafferUI/UIEditor.py

```diff
@@ -110,6 +110,17 @@ def __init__( self, scriptNode, **kw ) :
 						MetadataWidget.FileSystemPathMetadataWidget( key = "icon" )
 					)
 
+				with _Row() as self.__plugAddButtons :
+
+					_Label( "Add Plugs" )
```

Comment:
The grammar feels a bit off to me here : labels are usually nouns, but this is an imperative like you might have on a button. Internally they're called "Plug Adders", is that too awkward? Maybe "Plug Creators"? @medubelko, am I barking up the wrong tree?

Reply:
@medubelko Any thoughts on this, or I'll go for "Plug Creators?

Reply:
Sorry for the late response. Lost in the heap.

"Plug Creators" sounds like they're going to generate plugs on their own in some way. If "Plug Adders" are out, what about "Addable Plugs"?

Reply:
No worries, thanks @medubelko. They do create plugs, when you drag on to them with a connection, so thats probably a good name then?

Reply:
@medubelko We wanted to get a bug fix today, so we went for "Plug Creators" for now (as thats what they do), more than happy to change this for the next release if desired.